tools/cmake/Makefile

```diff
@@ -29,7 +28,20 @@ HOST_CONFIGURE_VARS += \
 
 HOST_CONFIGURE_ARGS := \
 	$(if $(MAKE_JOBSERVER),--parallel="$(MAKE_JOBSERVER)") \
-	--prefix=$(STAGING_DIR_HOST)
+	--prefix=$(STAGING_DIR_HOST) \
```

Comment:
I wonder if we should start putting quotes around this to eventually end up with a build system that accepts whitespaces

Reply:
Done.